src/baseclasses/chemical_energy/__init__.py

```diff
@@ -16,32 +16,64 @@
 # limitations under the License.
```

Comment:
Theses changes are just linting right?

Reply:
Yes all linting and I added one row `from .general_measurement import GeneralMeasurement`